pkg/internal/reporter/reporter_test.go

```diff
@@ -52,12 +52,12 @@ func TestReportShouldSucceed(t *testing.T) {
 
 func TestReportShouldSuccessfullyReportResults(t *testing.T) {
 	const (
-		expectedTrafficGenSentPackets        = 0
-		expectedTrafficGenOutputErrorPackets = 0
-		expectedTrafficGenInputErrorPackets  = 0
-		expectedVMUnderTestReceivedPackets   = 0
-		expectedVMUnderTestRxDroppedPackets  = 0
-		expectedVMUnderTestTxDroppedPackets  = 0
+		expectedTrafficGenSentPackets        = 100
+		expectedTrafficGenOutputErrorPackets = 11
+		expectedTrafficGenInputErrorPackets  = 12
+		expectedVMUnderTestReceivedPackets   = 80
+		expectedVMUnderTestRxDroppedPackets  = 13
+		expectedVMUnderTestTxDroppedPackets  = 14
```

Comment:
These values are used as the expected values in the successful scenario, which differs from what we consider a successful test.

Could you please use realistic results in the successful scenario?

Reply:
refactored the whole thing so that we could have real results for all results scenarios.